packages/contracts-bedrock/deploy-config/devnetL1.json

```diff
@@ -45,6 +45,6 @@
   "l2GenesisRegolithTimeOffset": "0x0",
   "faultGameAbsolutePrestate": "0x41c7ae758795765c6664a5d39bf63841c71ff191e9189522bad8ebff5d4eca98",
   "faultGameMaxDepth": 30,
-  "faultGameMaxDuration": 300,
```

Comment:
Why this bump?

Reply:
The previous 5 minute limit with a game depth of 30 only gave about 10 seconds per move, so e2e tests could fail because the clock expired.  Adding gzip saves disk space but uses more CPU so makes things a bit slower when under the load of e2e tests.